src/Target.php

```diff
@@ -72,6 +73,11 @@ class Target extends \yii\log\Target
     ];
 
     /**
+     * @var int max character in message text.
+     */
+    public $messageMaxLength = 3000;
```

Comment:
I think it's better to call this `substitutionMaxLength` because it only limits a single substitution max value length.